seed_services_cli/identity_store.py

```diff
@@ -29,10 +29,12 @@ def search(ctx, address_type, address):
         raise click.UsageError(
             "Please specify address type and address. See --help.")
     click.echo("Looking for %s of %s." % (address_type, address))
-    results = api.get_identity_by_address(address_type, address)
-    click.echo("Found %s results:" % results["count"])
-    if results["count"] > 0:
-        for result in results["results"]:
+    results = list(
+        api.get_identity_by_address(address_type, address)['results'])
+    num_results = len(results)
+    click.echo("Found %s results:" % num_results)
```

Comment:
Same thing here about the if statement not really being needed, you could put the `len()` in the `echo` and remove the `if`.